### DISCUSSION TITLE
How to add Gopher and Gemini link support?

### DISCUSSION THREAD

**Discussion by frrobert2** [datenstrom/community#595] [Ask a question] [answered]
Just an FYI for a very very small minority.

If you want to create a link for either a Gopher or Gemini site you need to the edit the file system/extensions/core.php

Replace all occurrences of `http|https|ftp|mailto` with `http|https|ftp|mailto|gopher|gemini` in core.php

This allows links starting with either gopher or gemini to be treated as valid.

**Comment by annaesvensson** [comment, marked as answer]
Thank you. 👍 Marking as answered/solved. See also #1032.
